frame/broker/src/coretime_interface.rs

```diff
@@ -52,7 +52,7 @@ pub trait CoretimeInterface {
 }
 impl CoretimeInterface for () {
 	type AccountId = ();
-	type Balance = u64;
+	type Balance = u128;
```

Comment:
???

Reply:
This was done to integrate with kitchensink, but still use Identity for `ConvertBalance`. Will move it back and instead use a converter there.